components/IndoorDirections/TypesOfDirections/DifferentBuildingDirections.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Line, G } from 'react-native-svg';
+import { ClassGraph } from '../../../constants/ClassGraph';
+import { dijkstra, getFloorNumber, changeClassName } from '../Dijkstra/DijkstraAlgorithm';
+
+export function DifferentBuildingDirections(props) {
+    const rooms = props.rooms;
+    const graph = ClassGraph();
+    var goTo = props.from;
+    var tempGoTo = props.from;
+
+    if (props.from.includes(" ")) {
+        tempGoTo = goTo;
+        goTo = changeClassName(props.to);
+    }
+
+    if (props.to.includes(" ")) {
+        tempGoTo = goTo;
+        goTo = changeClassName(props.from);
+    }
+
+    const pathToElevator = dijkstra(graph, goTo, "elevator").path;
+    const pathToClass = dijkstra(graph, "elevator", "exit").path;
+
+    var getNextRoomElevator = (index) => {
+        if (index < pathToElevator.length){
+            return pathToElevator[++index];
+        }
+    }
+
+    var getNextRoomClass = (index) => {
+        if (index < pathToClass.length){
+            return pathToClass[++index];
+        }
+    }
+
+    const linesToElevator = [];
```

Comment:
is this responsible for drawing the physical lines from point A to the elevator?